src/lib/components/Segment.svelte

```diff
@@ -105,8 +105,6 @@
     border-radius: var(--border-radius);
     overflow: hidden;
 
-    width: 100%;
-
     @include media.min-width(medium) {
       width: var(--segment-min-width, var(--segment-width, fit-content));
```

Comment:
On desktop we set a special width? Don't we need any kind of width on mobile?

Reply:
Let's set it explicitly w/ css-vars https://github.com/dfinity/nns-dapp/pull/4674